packages/desktop/views/dashboard/wallet/views/ImplicitAccountCreationView.svelte

```diff
@@ -22,6 +23,7 @@
             alert: true,
         })
         $dashboardRouter.goTo(DashboardRoute.AccountManagement)
+        addToVisitedTabs(DashboardRoute.AccountManagement)
```

Comment:
I'm not sure if this would ever be executed. I would say update the store first and then redirect